.appveyor.yml

```diff
@@ -14,7 +14,6 @@ environment:
         PIP_DEPENDENCIES: gsd==1.9.3 duecredit
         DEBUG: "False"
         MINGW_64: C:\mingw-w64\x86_64-6.3.0-posix-seh-rt_v5-rev1\mingw64\bin
-        OPENBLAS_64: https://3f23b170c54c2533c070-1c8a9b3114517dc5fe17b7c3f8c63a43.ssl.cf2.rackcdn.com/openblas-5f998ef_gcc7_1_0_win64.zip
```

Comment:
We shouldn't need this---it is almost certainly left-over from when I copied the upstream SciPy appveyor config file as a template, but didn't know enough to remove this. The OpenBLAS runtime library should be provided in the binaries we pull in for NumPy/SciPy or through conda.